src/modules/Channel/context/ChannelProvider.tsx

```diff
@@ -152,8 +154,9 @@ interface ChannelProviderInterface extends ChannelContextProps, MessageStoreInte
   resendMessage(failedMessage: SendableMessageType): Promise<SendableMessageType>,
   // TODO: Good to change interface to using params / This part need refactoring
   sendMessage(props: SendMessageParams): Promise<UserMessage>,
-  sendFileMessage(file: File, quoteMessage: SendableMessageType): Promise<FileMessage>,
+  sendFileMessage(file: File, quoteMessage?: SendableMessageType): Promise<FileMessage>,
   sendVoiceMessage: (file: File, duration: number, quoteMessage?: SendableMessageType) => void,
+  sendMultipleFilesMessage: (files: Array<File>, quoteMessage?: SendableMessageType) => Promise<MultipleFilesMessage>,
```

Comment:
return type is different than `useSendMultipleFilesMessage()`. It's void there.

Reply:
Oh, it's a great catch! It seems the other `sendMessage` and `sendFileMessage` also have kept wrong return types!